docs/SETUP.md

```diff
@@ -0,0 +1,227 @@
+# Quickstart
+
+Make a copy of `.env_sample` called `.env` in the root of the project directory. Confirm settings,storage,SSL and ports.
+
+Run the dev_setup script to install all dependencies and create a new virtual environment (venv) for CodaLab:
+
+1. **Windows Users** do 
+    `cd codalab`
+    `.\dev_setup.bat`
+
+2. **Linux Users** do 
+    `cd codalab`
+    `source ./dev_setup.sh`
+
+Activate your environment:
+
+1. **Windows Users** do 
+    `venv\Scripts\activate`
+
+2. **Linux Users** do 
+    `source venv/bin/activate`
+
+Navigate to the root of the project directory, or where `docker-compose.yml` lives, and do:
+
+`docker-compose up -d`
+
```

Comment:
personally, I like seeing the commands inside of a virtualenv whenever i'm in a virtual env. But that's just a personal preference.  for instance, `(codalabvenv)$ docker-compose up -d`